yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -1036,6 +1211,9 @@ func getYBSessionInitScript(tconf *TargetConf) []string {
 	if checkSessionVariableSupport(tconf, SET_DEFAULT_ISOLATION_LEVEL_REPEATABLE_READ) {
 		sessionVars = append(sessionVars, SET_DEFAULT_ISOLATION_LEVEL_REPEATABLE_READ)
 	}
+	if checkSessionVariableSupport(tconf, SET_YB_FAST_PATH_FOR_COLOCATED_COPY) {
```

Comment:
shouldn't we only set this if `on-primary-key-conflict = IGNORE` (i.e. we're going with non-txn COPY)

Reply:
Good catch. Better to use it when required.
Although it should not cause any issue if transaction is used(no fast path in that case)

Reply:
done